zstash/hpss.py

```diff
@@ -95,7 +95,14 @@ def hpss_transfer(
             # Transfer file using `hsi`
             command: str = 'hsi -q "cd {}; {} {}"'.format(hpss, transfer_command, name)
             error_str: str = "Transferring file {} HPSS: {}".format(transfer_word, name)
-            run_command(command, error_str)
+            if transfer_type == "get":
+                try:
+                    run_command(command, error_str)
+                except RuntimeError:
+                    # Retry if `hsi get` fails.
+                    run_command(command, error_str)
```

Comment:
Does it make sense to always retry `hsi` get` in the event of failure? Or do we indeed want a command line option to request this?